recipe/meta.yaml

```diff
@@ -21,30 +23,89 @@ requirements:
-
-test:
-  requires:
-    - {{ compiler("c") }}
-  files:
-    - test_openlibm.c
-  commands:
-    - test -d "${PREFIX}/include/openlibm"                         # [unix]
-    - test -f "${PREFIX}/lib/libopenlibm.a"                        # [unix]
-    - test -f "${PREFIX}/lib/libopenlibm${SHLIB_EXT}"              # [unix]
-    - if not exist "%LIBRARY_BIN%\\libopenlibm.dll" exit 1         # [win]
-    - if not exist "%LIBRARY_INC%\\openlibm\\openlibm.h" exit 1    # [win]
-    - if not exist "%LIBRARY_LIB%\\libopenlibm.lib" exit 1         # [win]
-    - if not exist "%LIBRARY_LIB%\\libopenlibm_static.lib" exit 1  # [win]
-    - echo on  # [win]
-    - cl /Tc test_openlibm.c /I %CONDA_PREFIX%\Library\include\openlibm /link /LIBPATH:"%CONDA_PREFIX%\Library\lib" libopenlibm_static.lib /out:test_openlibm.exe  # [win]
-    - test_openlibm.exe  # [win]
-    - ${CC} -o test_openlibm test_openlibm.c -I${CONDA_PREFIX}/include/openlibm -L${CONDA_PREFIX}/lib -lopenlibm ${CFLAGS} ${LDFLAGS}  # [unix]
-    - ./test_openlibm  # [unix]
+outputs:
+
+  - name: openlibm
+    build:
+      run_exports:
+        - {{ pin_subpackage('libopenlibm' ~ so_major_version) }}  # [unix]
+        - {{ pin_subpackage('libopenlibm', max_pin='x.x') }}      # [win]
+    files:
+      - include/openlibm  # [unix]
+      - lib/pkgconfig/openlibm.pc  # [unix]
+      - lib/libopenlibm${SHLIB_EXT}  # [unix]
+      - Library\\include\\openlibm  # [win]
+      - Library\\lib\\pkgconfig\\openlibm.pc  # [win]
+    requirements:
+      build:
+        - {{ compiler("c") }}  # [not win]
+        - {{ compiler("m2w64_c") }}  # [win]
+      run:
+        - {{ pin_subpackage('libopenlibm' ~ so_major_version, exact=True) }}  # [unix]
+        - {{ pin_subpackage('libopenlibm', exact=True) }}  # [win]
+    test:
+      requires:
+        - {{ compiler("c") }}
+      files:
+        - test_openlibm.c
+      commands:
+        - test -d "${PREFIX}/include/openlibm"                         # [unix]
+        - test ! -f "${PREFIX}/lib/libopenlibm.a"                      # [unix]
+        - test -f "${PREFIX}/lib/libopenlibm${SHLIB_EXT}"              # [unix]
+        - if not exist "%LIBRARY_INC%\\openlibm\\openlibm.h" exit 1    # [win]
+        - if exist "%LIBRARY_LIB%\\libopenlibm_static.lib" exit 1      # [win]
+        - if not exist "%LIBRARY_BIN%\\libopenlibm.dll" exit 1         # [win]
+        - if not exist "%LIBRARY_LIB%\\libopenlibm.lib" exit 1         # [win]
+        - echo on  # [win]
+        - cl /Tc test_openlibm.c /I %CONDA_PREFIX%\Library\include\openlibm /link /LIBPATH:"%CONDA_PREFIX%\Library\lib" libopenlibm.lib /out:test_openlibm.exe  # [win]
+        - test_openlibm.exe  # [win]
+        - ${CC} -o test_openlibm test_openlibm.c -I${CONDA_PREFIX}/include/openlibm -L${CONDA_PREFIX}/lib -lopenlibm ${CFLAGS} ${LDFLAGS}  # [unix]
+        - ./test_openlibm  # [unix]
+
+  - name: libopenlibm{{ so_major_version }}  # [unix]
+  - name: libopenlibm                        # [win]
```

Comment:
I think the linter is mistaken because of how I have defined the libopenlibm packages on windows vs unix.